tests/cli/test_deploy.py

```diff
@@ -293,7 +293,7 @@ async def test_project_deploy(self, project_dir, prefect_client):
         assert deployment.work_pool_name == "test-pool"
         assert deployment.version == "1.0.0"
         assert deployment.tags == ["foo-bar"]
-        assert deployment.concurrency_limit == 42
```

Comment:
Note that you can still access `deployment.concurrency_limit` but it's now deprecated so it emits a warning on access and also it will be `None`.